org.gnome.clocks.json

```diff
@@ -34,8 +34,8 @@
             "sources": [
                 {
                     "type": "archive",
```

Comment:
Make sure you copy the correct config-options btw

Reply:
Nice catch updated the rest.